app/src/main/java/net/bible/android/view/activity/page/BibleView.kt

```diff
@@ -686,7 +685,7 @@ class BibleView(val mainBibleActivity: MainBibleActivity,
             "?lang=$lang&fontModuleNames=$fontModuleNames&styleModuleNames=$styleModuleNames&featureModuleNames=$featureModuleNames&rtl=$isRtl&night=$nightMode")
     }
 
-     fun onEvent(e: ReloadAddonsEvent) {
+     fun onEvent() {
```

Comment:
I think this change is unintentional? Breaks reloading addons...

Reply:
```suggestion
     fun onEvent(e: ReloadAddonsEvent) {
```

Reply:
> I think this change is unintentional? Breaks reloading addons...
Yeah, good find. Thanks.